pkgs/development/python-modules/swagger-ui-bundle/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchPypi, jinja2, flake8 }:
+
+buildPythonPackage rec {
+  pname = "swagger-ui-bundle";
+  version = "0.0.2";
+
+  src = fetchPypi {
+    pname = "swagger_ui_bundle";
+    inherit version;
+    sha256 = "0kk6y8p1fhmnjnxfafdw6dm1xc5wj8101y5h7hcy3rb2bxc1i94q";
+  };
+
+  propagatedBuildInputs = [ jinja2 flake8 ];
```

Comment:
I think `flake8` should be a `nativeBuildInputs` because it's in `setup_requires`.